docs/release.md

```diff
@@ -39,29 +39,59 @@ These artifacts are provided in their respective channels and their natural way
 of distribution.
 
 # Cadence and Timeline
-Both a new release branch and initial release candidate are cut on the **first
-business day of each month.**
-
-If no blocker issues are discovered the release candidate is **promoted to a full
-release after 5 business days.**
+New release branches are cut 3 times a year, so a release cycle will occupy about 15 weeks.
+This is the same release cycle as Kubernetes, but KubeVirt will trail Kubernetes releases
+by about a week:
+| Week Number in Year | K8s Release Number | K8s Release Week | KubeVirt Release Number | KubeVirt Release Week |                     Note                    |
+|:-------------------:|:------------------:|:----------------:|-------------------------|-----------------------|:-------------------------------------------:|
+| 1                   | 1.24               | 1 (January 03)   | 0.55                    | 2 (January 10)        |                                             |
+| 15                  | 1.24               | 15 (April 12)    | 0.55                    | 16 (April 19)         |                                             |
+| 17                  | 1.25               | 1 (May 23)       | 0.56                    | 2 (May 30)            | KubeCon + CloudNativeCon EU likely to occur |
+| 32                  | 1.25               | 15 (August 22)   | 0.56                    | 16 (August 29)        |                                             |
+| 34                  | 1.26               | 1 (September 5)  | 0.57                    | 2 (September 12)      | KubeCon + CloudNativeCon NA likely to occur |
+| 49                  | 1.26               | 14 (December 05) | 0.57                    | 15 (December 12)      |                                             |
+
+Initial release candidates are cut **every 4 weeks.**
+With a 15 week cadence, there will be at least 3 release candidates:
+- v*-rc.0
+- v*-rc.1
+- v*-rc.2
```

Comment:
I'd assume that rc.2 could eventually be the release asset?

Reply:
Yes, unless there is a blocker